extensions/amp-accordion/1.0/base-element.js

```diff
@@ -219,7 +219,15 @@ function ContentShimWithRef(
   const contentRef = useRef();
   contentRef.current = contentElement;
   useSlotContext(contentRef);
-  useImperativeHandle(ref, () => contentElement, [contentElement]);
+  useImperativeHandle(
```

Comment:
We could just do `ref.current = contentElement`, to avoid the weird sometimes imperative sometimes dom check.

Reply:
Updated!  Thanks!